.travis.yml

```diff
@@ -20,6 +23,12 @@ before_install:
   - export PATH=/home/travis/miniconda3/bin:$PATH
   # Update conda itself
   - conda update --yes conda
+  # test that npm exists with language: node_js specified.
+  # node-qunit-puppeteer is a runner.js like interface for
+  # headless-chrome and qunit.
+  # node-qunit-puppeteer installs its own chromium as a dependency.
+  # test forgoing of chrome as an addon.
+  - npm install -g node-qunit-puppeteer
```

Comment:
The comment "test forgoing of chrome as an addon" is also opaque to me; please clarify.  
Also, "foregoing" not "forgoing" :D

Reply:
Another 'note-taking' comment made during development - I'll replace with, "hence, a separate chrome installation is not required." I was installing chrome as an individual package. I removed it when I discovered node-qunit-puppeteer installs its own chromium.

Sorry, ran out of 'e's on Friday. :D

Reply:
> ran out of 'e's on Friday

 Bahahahahaha! Excellent :)